leanpkg/leanpkg/main.lean

```diff
@@ -45,22 +45,21 @@ assg ← solve_deps d,
 path_file_cnts ← mk_path_file <$> construct_path assg,
 write_file "leanpkg.path" path_file_cnts
 
-
-def make : io unit := do
+def make (lean_args : list string) : io unit := do
 manifest ← read_manifest,
 exec_cmd {
   cmd := "lean",
   args := (match manifest.timeout with some t := ["-T", repr t] | none := [] end) ++
-    ["--make"] ++ manifest.effective_path,
+    ["--make"] ++ manifest.effective_path ++ lean_args,
   env := [("LEAN_PATH", none)]
 }
 
-def build := configure >> make
+def build (lean_args :list string) := configure >> make lean_args
```

Comment:
Thanks. Sorry I missed the space -- the `S-SPC` key-binding in Emacs eats some of my spaces (I change it to TAB on my branches).